enclave-tls/src/lib/enclave_quotes/null/init.c

```diff
@@ -0,0 +1,15 @@
+#include <enclave-tls/log.h>
+#include <enclave-tls/err.h>
+#include <enclave-tls/enclave_quote.h>
+
+/* *INDENT-OFF* */
+enclave_quote_err_t null_init(enclave_quote_ctx_t *ctx,
+			      enclave_tls_cert_algo_t algo)
+{
+	ETLS_DEBUG("enclave_quote_null init() is called\n");
+
+	ctx->quote_private = "null";
```

Comment:
It is better to define a dummy global variable use to hold the so-called private quote data.